images/tier1/add-to_lor.svg

```diff
@@ -0,0 +1,6 @@
+<svg xmlns="http://www.w3.org/2000/svg" height="18" width="18" viewBox="0 0 18 18" mirror-rtl="true">
```

Comment:
Maybe even `lor-add` so that `lor-publish` could be grouped right with it alphabetically.